src/core/webidl.js

```diff
@@ -234,7 +234,7 @@ function typeIsWhitespace(webIdlType) {
 }
 
 const extenedAttributesLinks = new Map([
-  ["CEReactions", "HTML51#cereactions"],
+  ["CEReactions", "!custom-elements#cereactions"],
```

Comment:
(I don't mind either way, but chose `custom-elements` based on what was done in bdb2df1e6dad5256078e4bda9345c9bdf3fe0e2b)